assets/locales/service.cy.toml

```diff
@@ -287,9 +287,9 @@ description = "Download"
 one = "download,"
 
 [DownloadsReady]
-description = "Your files are being created. Please <a href=\"{{.URI}}\">refresh</a> the page."
-one = "Your file is being created. Please <a href=\"{{.arg0}}\">refresh</a> the page."
-other = "Your files are being created. Please <a href=\"{{.arg0}}\">refresh</a> the page."
+description = "<strong>Your files are being created for download.</strong><br />If the page doesn't automatically update <a href=\"{{.URI}}\">reload the page</a>."
```

Comment:
Trailing slashes/self closing elements are not needed; https://github.com/validator/validator/wiki/Markup-%C2%BB-Void-elements#trailing-slashes-in-void-element-start-tags-do-not-mark-the-start-tags-as-self-closing